firestore.rules

```diff
@@ -32,6 +32,10 @@ service cloud.firestore {
       allow update: if isSignedIn();
       allow delete: if isSignedIn() && request.auth.uid in resource.data.access.admin;
     }
+
+    match /projects/{project}/{monitor} {
```

Comment:
I think this can be nested under the `projects` section. Will need similar rules to `projects` also for each action